### PR TITLE
Update cowbird version from 2.2.0 to 2.3.0

### PR DESCRIPTION
**Pull request by Nazim-crim**
## Overview

Update cowbird version

## Changes

**Non-breaking changes**

- Update `cowbird` service from [2.2.0](https://github.com/Ouranosinc/cowbird/tree/2.2.0) to [2.3.0](https://github.com/Ouranosinc/cowbird/tree/2.3.0).

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -15,6 +15,11 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
+[//]: # (list changes here, using '-' for each new entry, remove this when items are added)
+
+[1.41.0](https://github.com/bird-house/birdhouse-deploy/tree/1.41.0) (2023-11-30)
+------------------------------------------------------------------------------------------------------------------
+
```
Comment (tlvu) [inline CHANGES.md:22]: Empty CHANGES.md ?!

Comment (Nazim-crim) [reply CHANGES.md:22]: I think there was an update and version bbump that happened at the same time of my change and the merge went wrongly. I've reverted my change and re bumped  version. It should be fixed. Can you confirm ?

Comment (tlvu) [reply CHANGES.md:22]: Yes, this time the diff looks good.

**Comment by Nazim-crim** [comment]
run tests

File: CHANGES.md (hunk 1 above)
Comment (Nazim-crim) [reply CHANGES.md:22]: okay I will merge once the tests finish running

**Review by tlvu — APPROVED** [review]

**Pull request merged**